server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -718,3 +728,18 @@ def _plain_to_html(self, text):
 
     def _get_cv(self, _id):
         return superdesk.get_resource_service('vocabularies').find_one(req=None, _id=_id)
+
+    def _get_country(self, country_code):
+        if not self._countries:
+            self._countries = self._get_cv('country').get('items', [])
+
+        country_keyword = [
+            c for c in self._countries
+            if c.get('qcode') == 'country_' + country_code.lower() and c.get('is_active')
+        ]
+        # add scheme and remove is_active
+        for c in country_keyword:
+            c['scheme'] = 'country'
+            if 'is_active' in c:
+                c.pop('is_active')
```

Comment:
I think here it modifies the item in `self._countries` so next time `is_active` won't be there
for the previous filter `and c.get('is_active')`. would be better to return a copy to avoid it:
```python
return [{'name': c['name'], 'qcode': c['qcode'], 'scheme': 'country'} for c in country_keyword]
```